.nvmrc

```diff
@@ -1 +1 @@
-16
+16.18.1
```

Comment:
might as well bump to v18 as that's the node version our vercel deployment is using